pkg/kyq/kyq.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2021 The Kubernetes Authors.
```

Comment:
The `yq` package is unnecessary. Please remove.

Reply:
Discussion of jsonpath vs yq is now pubilc:
https://docs.google.com/document/d/1SChsQ9-l13Czpq4YGq7HA2JorZjL5HQ-27CJzwX77yE/edit#

The reason we can't just use the JSONPath lib in client-go is that it doesn't support updates, only retrieval.

Reply:
switched to jsonpath